tests/integration/api/test_vector_aggregation_operations.py

```diff
@@ -57,27 +119,38 @@ def apply_function(xs):
     return out
 
 
-def test_random(data_with_array):
+@pytest.fixture(name="vector_user_entity", scope="module")
```

Comment:
we recreate a couple of entities here, as otherwise, there's some conflicts with the entity relationship hierarchy that conflicts w/ the non vector event/item tables.